src/3d/framegraph/qgsdebugtexturerenderview.cpp

```diff
@@ -0,0 +1,59 @@
+/***************************************************************************
+  qgsdebugtexturerenderview.cpp
+  --------------------------------------
+  Date                 : June 2024
+  Copyright            : (C) 2024 by Benoit De Mezzo
+  Email                : benoit dot de dot mezzo at oslandia dot com
+ ***************************************************************************
+ *                                                                         *
+ *   This program is free software; you can redistribute it and/or modify  *
+ *   it under the terms of the GNU General Public License as published by  *
+ *   the Free Software Foundation; either version 2 of the License, or     *
+ *   (at your option) any later version.                                   *
+ *                                                                         *
+ ***************************************************************************/
+
+#include "qgsdebugtexturerenderview.h"
+#include <Qt3DRender/QCamera>
+#include <Qt3DRender/QLayerFilter>
+#include <Qt3DRender/QLayer>
+#include <Qt3DRender/QRenderTargetSelector>
+#include <Qt3DRender/QRenderTarget>
+#include <Qt3DRender/QTexture>
+#include <Qt3DRender/QClearBuffers>
+#include <Qt3DRender/qsubtreeenabler.h>
+#include <Qt3DRender/QCameraSelector>
+#include <Qt3DRender/QRenderStateSet>
+#include <Qt3DRender/QDepthTest>
+#include <Qt3DRender/QCullFace>
```

Comment:
many of the includes are not needed (QCamera, QRenderTarget, QRenderTargetSelector, QTexture, QSubtreeEnabler, QCameraSelector)

Reply:
fixed

Reply:
`#include <Qt3DRender/qsubtreeenabler.h>` is still there though